AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -99,8 +115,17 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
                 rendererView!.leftAnchor.constraint(equalTo: previewView.leftAnchor).isActive = true
                 rendererView!.rightAnchor.constraint(equalTo: previewView.rightAnchor).isActive = true
                 loadingView.stopAnimating()
+                print("Lobby video view renderer successfully created.")
             } catch {
-                print("Failed to create renderer for lobby video view")
+                print("Failed to create renderer for lobby video view. " + (retry > 0 ? "Retry attempt #\(retry)" : "Attempt to retry..."))
```

Comment:
Since the denied to join is handled in `promptForTeamsAdmissionDenied` When will this retry be triggered?

Reply:
`promptForTeamsAdmissionDenied` leads to `resetRendererView()` being invoked. Within it there's `setupPreviewView` that contains the retry.

Reply:
I mean under what condition will it go into the catch block?

Reply:
The following statement: `previewRenderer = try VideoStreamRenderer(localVideoStream: localVideoStream)`
`VideoStreamRenderer` is from the Calling SDK

Reply:
When the camera is not ready, instantiating `VideoStreamRenderer` throws an error

Reply:
To be specific here's the error log:
`Error Domain=ACSException Code=64 "Cannot create multiple renderers for same device or stream."`

Reply:
Sorry, But I wonder if it's necessary to set the retrySet to retry the VideoStremRender

I could image the reason why   `previewRenderer = try VideoStreamRenderer(localVideoStream: localVideoStream)` failed is because the local render view inside CallingView is not disposed immediately when when user been kick out from CallingVC. Is that possible that we  manual call local video render dispose function inside `CallingViewController` when it's being dismissed, like the function (ViewWillDisappear)

Reply:
Actually that helps! Immediately disposing the renderer for local participant view upon leaving the `CallingViewController` does the job.
Removed the retry mechanism since no longer needed.

Reply:
Sounds good!